Makefile

```diff
@@ -98,16 +102,21 @@ e2e-test: ## Run E2E Playwright tests in a Docker container and copy the report
 e2e-test: e2e-build
 	@:$(call check_defined, APP_NAME, You must pass in a specific APP_NAME)
 	@:$(call check_defined, BASE_URL, You must pass in a BASE_URL)
-	docker run --rm \
+	docker run --rm\
 		--name playwright-e2e-container \
 		-e APP_NAME=$(APP_NAME) \
 		-e BASE_URL=$(BASE_URL) \
+		-e CURRENT_SHARD=$(CURRENT_SHARD) \
+		-e TOTAL_SHARDS=$(TOTAL_SHARDS) \
+		-e CI=$(CI) \
 		-v $(PWD)/e2e/playwright-report:/e2e/playwright-report \
+		-v $(PWD)/e2e/blob-report:/e2e/blob-report \
 		playwright-e2e
```

Comment:
this change should have been part of the first e2e sharding PR, let's address that discrepancy before continuing on this PR review

Reply:
yep - outdated as of now